src/Server/TextDocument.php

```diff
@@ -80,12 +62,12 @@ class TextDocument
     protected $composerLock;
 
     /**
-     * @param PhpDocumentLoader  $documentLoader
+     * @param PhpDocumentLoader $documentLoader
      * @param DefinitionResolver $definitionResolver
-     * @param LanguageClient     $client
-     * @param ReadableIndex      $index
-     * @param \stdClass          $composerJson
-     * @param \stdClass          $composerLock
+     * @param LanguageClient $client
+     * @param ReadableIndex $index
+     * @param \stdClass $composerJson
+     * @param \stdClass $composerLock
```

Comment:
I don't know if Sara ran a formatter or something that undid the alignment. I can revert it if you prefer.